File: Módulo Python/Códigos/3 - Estrutura de Decisão/Exercicio.25.py
#25. Faça um algoritmo para ler três números e ordene-os em ordem decrescente.

num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))
num3 = float(input("Digite o terceiro número: "))

maior = num1
meio = num2
menor = num3

if num2 > maior:
    maior = num2
    meio = num1
    menor = num3
if num3 > maior:
    maior = num3
    meio = num1
    menor = num2
if num3 > meio:
    meio = num3
    menor = num2

print("Os números em ordem decrescente são:", maior, meio, menor)

File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.25.java
// Faça um algoritmo para ler três números e ordene-os em ordem decrescente

import java.util.Scanner;

public class Exercicio25 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double n1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double n2 = scanner.nextDouble();

        System.out.print("Digite o terceiro número: ");
        double n3 = scanner.nextDouble();

        double menor = Math.min(Math.min(n1, n2), n3);
        double maior = Math.max(Math.max(n1, n2), n3);
        double meio = (n1 + n2 + n3) - menor - maior;

        System.out.println("Os números em ordem decrescente são " + maior + " , " + meio + " e " + menor);

        scanner.close();
    }
}